openpype/lib/build_template.py

```diff
@@ -0,0 +1,28 @@
+import avalon
+import importlib
+
+concrete_loaders_modules = {
+    'maya': 'openpype.hosts.maya.api.template_loader'
+}
+
+
+class BuildWorkfileTemplate:
```

Comment:
This does not need class at all. It is a function.

Reply:
solved